src/components/Books/BooksList.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const BooksList = () => {
```

Comment:
- [ ] Based on the requirements of this project, the BooksList component should take the book data as a prop and renders the title :+1: and author :angry: Let this component receive the book data as prop and equally make sure to render the author of the book.